app/views/admin/resources/update_quantity_error.js.erb

```diff
@@ -0,0 +1 @@
+$('#messages').html("<%= j render 'layouts/messages' %>");
```

Comment:
why not use unobtrusive flash? Then you shouldn't need this.
https://github.com/leonid-shevtsov/unobtrusive_flash